pkg/k8s/endpoints.go

```diff
@@ -414,7 +414,17 @@ func (es *EndpointSlices) GetEndpoints() *Endpoints {
 	allEps := newEndpoints()
 	for _, eps := range es.epSlices {
 		for backend, ep := range eps.Backends {
-			allEps.Backends[backend] = ep
+			// EndpointSlices may have duplicate addresses on different slices.
```

Comment:
It's worthwhile to add snippets of `kubectl get endpointslices` to explain the intention. Wdyt?

Reply:
yeah, I like that